esmvaltool/recipe_schema.yml

```diff
@@ -28,6 +28,8 @@ variable:
   reference_dataset: str(required=False)
   alternative_dataset: str(required=False)
   mip: str(required=False)
+  fx_mask: bool(required=False)
+  fx_variable: list(required=False)
```

Comment:
Just for my understanding, we explicitely need to specify which fx we want to have since they need to go through the CMOR check, right?